c/pthread-nondet/nondet-loop-bound_true-unreach-call.c

```diff
@@ -0,0 +1,27 @@
+#include <pthread.h>
+#include "assert.h"
+
+volatile int x;
+volatile int n;
+
+void* thr1(void* arg) {
+    assert(x <= n);
+}
+
+void* thr2(void* arg) {
+    int t;
+    t = x;
+    x = t + 1;
+}
+
+int main(int argc, char* argv[]) {
+    pthread_t t1, t2;
+    int i;
+    x = 0;
+    n = __VERIFIER_nondet_uint() % 20 + 20;
```

Comment:
Could we use a `__VERIFIER_assume(condition)` here instead to ensure the right values for n? Modulo handling might be complicated for verifiers/ SMT solvers, whereas a condition like `(20<=n && n<40)` might be more useful. The intention for this task sure is to check concurrency, not to see whether modulo is handled correctly.
I just checked, there are already some tasks in the master branch that also use modulo for setting the interval of a nondet value. @kfriedberger, @dbeyer, is this intentional / should this be corrected?

Reply:
I think modulo operations should be avoided if easily possible and not affecting the intention of the verification task, except in the 'bit-vector' category.

Reply:
@MartinSpiessl @dbeyer Could someone take a look at #599?